README.md

```diff
@@ -1,53 +0,0 @@
-#SF-WDI-23 Homework submission
-
```

Comment:
oh no!  you deleted the class README.md file!  

I'll fix it this time, but try using `git status` to see what you are committing.  In fact I really recommend doing `git status` before EVERY git command just for safety and so you get to know what you're doing with git.